eng/targets/Imports.targets

```diff
@@ -404,4 +404,10 @@
     </ItemGroup>
   </Target>
 
+  <!-- Override this to work around https://github.com/dotnet/sdk/issues/13427. The property doesn't
+       appear to be used for anything other than controlling the warning. -->
+  <PropertyGroup>
+    <_MicrosoftWindowsDesktopSdkImported>false</_MicrosoftWindowsDesktopSdkImported>
+  </PropertyGroup>
```

Comment:
📝 File a bug to fix this when we move to RC2, or submit a draft PR so it's easy to take whenever